app/liquid_tags/podcast_tag.rb

```diff
@@ -14,104 +15,30 @@ class PodcastTag < LiquidTagBase
   def initialize(_tag_name, link, _tokens)
     @episode = fetch_podcast(link)
     @podcast ||= Podcast.new
+    @style = render_style
```

Comment:
`render_style` could be inline style-tag in the erb file. Also, it doesn't need to be passed in ActionController since it mostly relies on `@podcast` object.

Reply:
changed to inline in recent commit